cfpb/cfgov-refresh
# Modifying heading labels for link blobs

Modifying heading labels for the half-width and third-width link blobs

## Changes

- Modified `cfgov/v1/atomic_elements/molecules.py` to change labels as follows:
 - `Heading` -> `H3 heading`
 - `Sub-Heading` -> `H4 heading`
 - `Sub-Heading icon` -> `H4 icon`

## Testing

- Add a `Browse page` via Wagtail.
- Add a `Half Width Link Blob Group` 
  - Fill in the `H3` field.
- Add a `Third Width Link Blob Group`
  - Fill in the `H4` field.
- Preview and verify the text hierarchy is as expected.

## Review

- @chosak 
- @richaagarwal 
- @schaferjh 


## Screenshots
<img width="1109" alt="screen shot 2016-09-22 at 11 44 17 am" src="https://cloud.githubusercontent.com/assets/1696212/18755451/4b7f6502-80ba-11e6-94d9-1e753c5cb70f.png">


## Checklist
